src/main/java/Assignment1.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Scanner;
+
+class ItemList {
```

Comment:
create Model class for ItemList (to follow MVC)

Reply:
done.